packages/esm-appointments-app/src/appointments/appointment-tabs.component.tsx

```diff
@@ -4,7 +4,6 @@ import { Tab, TabList, Tabs, TabPanel, TabPanels } from '@carbon/react';
 
 import { type ConfigObject } from '../config-schema';
 import { useConfig } from '@openmrs/esm-framework';
-import { useVisits } from '../hooks/useVisits';
```

Comment:
Instead of fetching the visits here and passing them down as a prop, I moved the fetch to the actions component, where they are actually used/